src/components/RequestRepository/AnnualEmail.tsx

```diff
@@ -198,62 +207,91 @@ const AnnualEmail = () => {
+          <Alert type="error" isClosable={false}>
+            {t('home:adminHome.GRT.requesterUpdateEmail.modal.error')}
+          </Alert>
+        )}
+
+        {!loading && !error && (
+          <>
+            <Form onSubmit={handleSubmit(onSubmit)} className="maxw-none">
+              {typedEntries(list).map(([key, statusLabel]) => (
+                <Controller
+                  key={key}
+                  name={key}
+                  control={control}
+                  render={({ field: { ref, ...field } }) => (
+                    <CheckboxField
+                      {...field}
+                      id={key as string}
+                      value={key as string}
+                      label={statusLabel}
+                      data-testid={`checkbox-${key}`}
+                    />
+                  )}
                 />
+              ))}
+
+              {warning && (
+                <Alert slim type="warning">
+                  {t('home:adminHome.GRT.requesterUpdateEmail.modal.noEmail')}
+                </Alert>
               )}
-            />
-          ))}
-
-          {warning && (
-            <Alert slim type="warning">
-              {t('home:adminHome.GRT.requesterUpdateEmail.modal.noEmail')}
-            </Alert>
-          )}
-
-          <ModalFooter>
-            <div className="display-flex flex-gap-3">
-              <Button
-                type="submit"
-                disabled={!hasSelected}
-                className="margin-y-0"
-              >
-                {t(
-                  'home:adminHome.GRT.requesterUpdateEmail.modal.openEmailButton'
-                )}
-              </Button>
-              <Button
-                type="button"
-                onClick={onCopy}
-                disabled={!hasSelected}
-                unstyled
-                className="margin-y-0"
-              >
-                {t(
-                  'home:adminHome.GRT.requesterUpdateEmail.modal.copyEmailButton'
-                )}
-              </Button>
-            </div>
-          </ModalFooter>
-        </Form>
+
+              <ModalFooter>
+                <div className="display-flex flex-gap-3">
+                  <Button
+                    type="submit"
+                    disabled={!hasSelected}
+                    className="margin-y-0"
+                  >
+                    {t(
+                      'home:adminHome.GRT.requesterUpdateEmail.modal.openEmailButton'
+                    )}
+                  </Button>
+                  {emailsCopied ? (
+                    <div className="display-flex flex-align-center flex-justify-space-between">
+                      <span className="text-italic text-success-dark">
+                        <Icon.Check className="margin-right-1" />
+                        {t(
+                          'home:adminHome.GRT.requesterUpdateEmail.modal.copied'
+                        )}
+                      </span>
+                    </div>
```

Comment:
The `display-flex flex-align-center flex-justify-space-between` classes on this `div` aren't center aligning the text and icon because they're wrapped in a `span` tag. You could probably just remove the `div` and move the classes to the `span` tag (or the other way around).